docs/examples/bci2000/connect.md

```diff
@@ -0,0 +1,133 @@
+# Connecting DSI Headset
+
+This guide will walk you through how to connect your Wearable Sensing DSI headset to BCI2000 for data acquisition.
+
+```{admonition} Note
+:class: sidebar note
+Please ensure BCI2000 is already installed on your system. If you haven't installed it yet, you can find the download instructions [here](https://www.bci2000.org/mediawiki/index.php/DownloadBCI2000).
+```
+
+## Update the DSI API Driver
+
+BCI2000 includes a community-contributed module called DSISerial, which supports Wearable Sensing DSI EEG systems. This module is already built into BCI2000, but you need to update its driver manually to ensure full compatibility.
+
+1. Download the Compatible Driver File
+
+   * Download the driver that was built alongside the current version of *DSISerial*. Although new versions of the Wearable Sensing *DSI API* exist, they have not yet been tested with *DSISerial* and may not work correctly.
+
+   * [DSI_API version 1.20.1](https://www.bci2000.org/svn/trunk/src/contrib/SignalSource/DSISerial/DSI_API/)
+
+   > Choose the `.dll` file that matches your system architecture.
+
+2. Locate and copy the `.dll` file.
+
+   * Open the unzipped folder and find the dynamic library file related to your system. This tutorial is using windows so I will copy the `.dll` with the name `libDSI-Windows-x86_64.dll`
+
+3. Paste into the `prog` directory.
+
+   * Locate where you installed the latest version of *BCI2000* and locate the `prog` directory. Inside `prog` paste your `.dll` file named `libDSI-Windows-x86_64.dll`. Your path should look similar to this:
+
+   ```sh
+   BCI2000\BCI2000 v3.6.beta.R7385\BCI2000.x64.bundled\prog
+   ```
+
+<!-- To access it, you will need to download the source code from the SVN repository:
+[http://www.bci2000.org/svn/trunk/src/contrib/SignalSource/DSISerial](http://www.bci2000.org/svn/trunk/src/contrib/SignalSource/DSISerial)
+
+```{admonition} Note
+:class: attention
+Accessing the DSISerial SVN repository requires an existing BCI2000 account and an SVN client, as these links cannot be downloaded directly via a browser.
+
+If you're unfamiliar with SVN tools, we recommend [TortoiseSVN](https://tortoisesvn.net/downloads.html) for Windows users, which provides a simple to use interface to download and manage SVN repositories.
+``` -->
+
+### Set the Environment Variable
+
+After downloading and placing the DSI API into the correct directory, you'll need to conifugre your environment variable so it points to the correct COM port for your DSI headset. This allows BCI2000 to automatically detect the device is the easiest method recommended by BCI2000.
+
+**Steps to Add the Environment Variable**
+
+1. Open the Windows Search Bar.
+
+2. Search "Edit the system environment Variables"
+
```

Comment:
A missing step:

3. A window with a title "System Properties" should open. Click the `Environment Variables...` button near the bottom. This will open another window.